docs/Bioinformatics-Skills/Simulate_Illumina_Reads.md

```diff
@@ -0,0 +1,337 @@
+
+=== "AWS Instance Code"
+
+    ```
+    for i in {1..n};
+    do iss generate --genomes ../reference_genome/GRCh38_latest_genomic.fna --model hiseq --n_reads 5M --cpus 4 --output ${i}_my_sim;
+    done
+    ```
+
+!!! Warning
+    Depending on how many times you run the simulation and/or how many reads you simulate, the code may take a really long time to complete! You can start out by running a small set of reads (2-3) to test the code before scaling up.
+
+
+## Quality control
+
+With real data, one must always perform quality control to ensure that the raw data looks good. With simulated data, you can do some simple quality control to make sure nothing went terribly wrong with the simulation itself and that you have the desired read length.
+
+Let's use FastQC to generate a quality control report.
+
+### Install FastQC
+
+To install FastQC run this code:
+
+=== "AWS Instance Code"
+
+    ```
+    sudo apt install fastqc
+    ```
+
+### Run FastQC
+
+Now run FastQC on your fastq files:
+
+=== "AWS Instance Code"
+
+    ```
+    cd fastq
+    fastqc *.fastq
+    ```
+You are using the `*` wildcard to help specify all of the .fastq files here.
+
+
+### Transfer to Local Computer
+
+The easiest way to visualize the output is to transfer it to your local computer. First, let's list all the files and pull out the report summaries (.html) using ls:
+
+=== "AWS Instance Code"
+
+    ```
+    ls *fastqc.zip
+    ls *.html
+    ```
+
+Now you can move the ".html" files to your local computer to visualize:
+
+=== "Local Machine Code"
+
+    ```
+    scp -i ~/Desktop/amazon.pem ubuntu@ec2-??-???-???-??.us-east-2.compute.amazonaws.com:/home/ubuntu/fastq/\*.html ~/Desktop/fastqc/.
+    ```
+
+=== "Expected Output"
+    You might need to confirm the download by typing "yes" if prompted:
+
+    ```
+    Are you sure you want to continue connecting (yes/no/[fingerprint])? yes"
+    ```
+
+!!! Important
+
+    This code block must be run on your local terminal window and not on the AWS terminal. You can start a new terminal by typing `cmd+N`
+
+    Remember to replace:
+
+    - `~/Desktop/amazon.pem` with the [path to your amazon.pem](GWAS-in-the-cloud/aws_instance_setup.md)
+
+    - `??-???-???-??.us-east-2.compute.amazonaws.com` with your specific instance. For more details on how to connect to an instance, visit the [AWS set up page of the "GWAS in the Cloud" tutorial](GWAS-in-the-cloud/download_accessAWS.md).
+
+
+Here is an image of the top of an example FastQC report.
+
+![](../images/Simulated_Data_Fastqc.png "FastQC report")
+
+
+### Interpretation
+
+**Watch this [YouTube video](https://www.youtube.com/watch?v=bz93ReOv87Y) on how to interpret FastQC results**:
+
+[![FastQC video tutorial](http://img.youtube.com/vi/bz93ReOv87Y/0.jpg)](https://www.youtube.com/watch?v=bz93ReOv87Y "FastQC Interpretation")
+
+All simulated reads are 126 bp long :)
```

Comment:
edit suggestion: "From the FastQC report, we see that the overall read quality scores are good and read length is 126bp. These simulated reads are now ready to be used for analysis!"